app/controllers/api/v1/activity_sessions_controller.rb

```diff
@@ -104,31 +104,29 @@ def concept_tag_result_allowed_keys
   # Alias the following request parameters:
   # Map each result to the following structure:
   # {
-  #   concept_tag: "Creative Writing",
-  #   concept_class: "Writing Concepts",
+  #   concept: "uid-abcde12345",
   #   student_input: "The dog jumped over the cat."
   # },
   # Becomes this:
   # {
   #   metadata: {
-  #     concept_tag: "Creative Writing",
-  #     concept_class: "Writing Concepts",
+  #     concept: "uid-abcde12345",
   #     student_input: "The dog jumped over the cat."
   #   }
   # }
   #
-  # concept_tag_results -> concept_tag_results_attributes
+  # concept_results -> concept_results_attributes
   def transform_incoming_request
```

Comment:
Yeah, I can change the request format so that it's something like this:

```
{
  concept_uid: "foobar",
  metadata: {
    correct: true,
    baz: 'bar'
  }
}
```
Does that seem overall better? @marcellosachs @wlaurance